backend-checks/src/test/scala/uk/gov/nationalarchives/tdr/localaws/backendchecks/checks/metadata/FakeFileFormatTest.scala

```diff
@@ -0,0 +1,42 @@
+package uk.gov.nationalarchives.tdr.localaws.backendchecks.checks.metadata
+
+import java.nio.file.Paths
+
+import org.scalatest.matchers.should
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.propspec.AnyPropSpec
+
+class FakeFileFormatTest extends AnyPropSpec with TableDrivenPropertyChecks with should.Matchers {
+
+  case class FileFormatExample(
+                                fileName: String,
+                                expectedFileFormatId: Option[String],
+                                expectedExtension: Option[String]
+                              )
+  val defaultFileFormat = "x-fmt/111"
+  val examples = Table(
+    "Antivirus metadata examples",
```

Comment:
Should read: "**File format** metadata examples"

Reply:
Fixed